site/templates/nav-banner.jet

```diff
@@ -0,0 +1,11 @@
+{{block navBanner()}}
+
+ {{showBanner:= config("show_nav_banner") == "true"}}
+
+ {{if showBanner}}
+  <div class="nav-banner">
```

Comment:
Can you fix the indentation please!

Reply:
Do you mean removing the spaces?

Reply:
Our indent style is 2 spaces... looks like you have 1 space except for where you have an extra space to mis-align the end of the block.